inc/order_supplier.class.php

```diff
@@ -28,9 +28,7 @@
  * -------------------------------------------------------------------------
  */
 
-if (!defined('GLPI_ROOT')) {
-    die("Sorry. You can't access directly to this file");
-}
+
 
 class PluginOrderOrder_Supplier extends CommonDBChild // phpcs:ignore
```

Comment:
You can remove `php_codesniffer` when you have `php-cs-fixer`, especially if they conflict with each other:
```suggestion
class PluginOrderOrder_Supplier extends CommonDBChild
```